README.md

```diff
@@ -0,0 +1,44 @@
+# Firmware for Signet Devices
+
+## Prerequisites
+
+#### Cloning Directions
+
+This project uses git submodules, you will either need to use `git clone
+--recursive <repourl>` or after cloning use `git submodule update --init`.
+
+#### Dependencies
+
+The following packages must be installed:
+
+`apt install build-essential autoconf automake gperf texinfo help2man ncurses-dev bison flex libtool-bin libjson-c-dev libiberty-dev`
+
+## Building Firmwares
+
```

Comment:
Probably should mention that you need a GNU/Linux system. It many be possible to pull it off under windows but I've never tried to do it.

Reply:
Done.

Reply:
Added one more dependency: dfu-util